control/plugin/rpc/plugin.go

```diff
@@ -90,7 +90,13 @@ func ToConfigPolicy(reply *GetConfigPolicyReply) *cpolicy.ConfigPolicy {
 			nodes[k] = cpolicy.NewPolicyNode()
 		}
 		for key, val := range v.Rules {
-			br, err := cpolicy.NewBoolRule(key, val.Required, val.Default)
+			var br *cpolicy.BoolRule
+			var err error
+			if val.HasDefault {
+				br, err = cpolicy.NewBoolRule(key, val.Required, val.Default)
+			} else {
+				br, err = cpolicy.NewBoolRule(key, val.Required)
+			}
 			if err != nil {
 				// The only error that can be thrown is empty key error, ignore something with empty key
```

Comment:
If we call the plugin's GetConfigPolicy and receive a ConfigPolicyNode which contains an empty key we might want to log this (level=warning).  What do you think?